### PR TITLE
Update Migrating_from_Klipper.md

### PR DESCRIPTION
**Pull request by BoredFishRE**
:)
Basically what's in readme with literally one minor change.

Adding migration instructions.

## Checklist

- [x] pr title makes sense
- [x] squashed to 1 commit
- [x] added a test case if possible
- [x] if new feature, added to the readme
- [x] ci is happy and green

### CODE REVIEW

File: docs/Migrating_from_Klipper.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+If desired, make a backup copy of your existing Klipper installation by running:
+
+```bash
+mv ~/klipper ~/klipper_old
+```
+
+Then clone the Kalico repo and restart the klipper service:
+
+```bash
+git clone https://github.com/kalico-crew/kalico.git ~/klipper
```
Comment (rogerlz) [inline docs/Migrating_from_Klipper.md:19]: Typo: https://github.com/KalicoCrew/kalico

File: docs/Migrating_from_Klipper.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+- Select 'Y' to apply the changes
+- Enter 'B' for back twice
+- 'Q' to quit
+
+### Option 3. Adding a git-remote to the existing installation
+Can switch back to mainline klipper at any time via a `git checkout upstream_main`
+
+```bash
+cd ~/klipper
+git remote add kalico https://github.com/kalico-crew/kalico.git
```
Comment (rogerlz) [inline docs/Migrating_from_Klipper.md:44]: here too

File: docs/Migrating_from_Klipper.md [hunk 3]
```diff
@@ -1,2 +1,50 @@
...
+
+For users that are not comfortable using Git directly, [KIAUH v6](https://github.com/dw-0/kiauh) is able to use custom repositories.
+
+To do this, add the Kalico repo to KIAUH's custom repository settings with the following steps:
+
+From the KIAUH menu select:
+
+- [S] Settings
+- 1\) Set custom Klipper repository
+- Use `https://github.com/kalico-crew/kalico` as the new repository URL
```
Comment (rogerlz) [inline docs/Migrating_from_Klipper.md:33]: here too

**Review by rogerlz — CHANGES_REQUESTED** [review]

**Comment by BoredFishRE** [comment]
oops hit the wrong button hang on

**Comment by BoredFishRE** [comment]
there we go should be good, squashed to 1 commit. Github makes this unnecessarily difficult

**Pull request merged**